### ISSUE TITLE
Add `electron-webpack test` command

### ISSUE THREAD

**Issue by amaury1093** [electron-userland/electron-webpack#266]
Right now, if I want to run tests, I need to:
- install jest (or another test framework)
- add manually a babel.config.js in the root folder, so that jest knows how to parse the tests

The 2nd step should be removed.

I'd suggest to do like react-scripts, add a `test` command, which will run jest with the correct babel config.

**Comment by thinkclay** [comment]
Agreed. It's way too painful trying to figure out where the test framework is slipping up. I recently swapped one of my projects from a fully custom setup to using electron-webpack (which went fairly smoothly) but had to do a lot of trial and error to finally discover that the stock tsconfig options don't play nice with Jest out of the box.

**Comment by beorn** [comment]
What is the correct way to configure Jest so it's most compatible with your electron-webpack settings?

**Comment by ejohnso49** [comment]
Would love to know as well, currently fighting with all of these components
